packages/web-api/src/response/SearchAllResponse.ts

```diff
@@ -140,6 +140,7 @@ export interface MessagesMatch {
   attachments?:  MatchAttachment[];
   is_mpim?:      boolean;
   score?:        number;
+  files?:        File[];
```

Comment:
Changes here aren't relevant to PR, but auto-generated by `generate-web-api-types.sh`
See also, `SearchMessagesResponse.ts`

Reply:
Yes, this is fine. We've detected this pattern and updated the type def very recently.